cpp/include/cudf/strings/repeat_strings.hpp

```diff
@@ -75,16 +82,55 @@ std::unique_ptr<string_scalar> repeat_strings(
  * out is ['aaaaaa', null, '', 'bbcbbcbbc']
  * @endcode
  *
- * @param input The column containing strings to repeat.
+ * @param strings The column containing strings to repeat.
```

Comment:
I would prefer to use `input` here instead of `strings`. There have been cases in the past where the compiler gets confused since `strings` is also the name of the namespace.